messages/cardano.proto

```diff
@@ -85,16 +92,22 @@ message CardanoSignTransactionRequest {
     repeated AssetGroup asset_groups = 4;
   }
 
-  // See https://github.com/input-output-hk/cardano-ledger-specs/blob/d0aa86ded0b973b09b629e5aa62aa1e71364d088/eras/alonzo/test-suite/cddl-files/alonzo.cddl#L150
+  // See https://github.com/IntersectMBO/cardano-ledger/blob/cardano-ledger-conway-1.12.0.0/eras/conway/impl/cddl-files/conway.cddl#L273
   message Certificate {
     message StakeDelegation {
-      repeated uint32 keypath = 1;
+      repeated uint32 stake_keypath = 1;
```

Comment:
Seems like an unrelated change that breaks downstream libraries. Is this necessary?

Reply:
It was suggested during internal review to make naming consistent and in vote delegation it makes sense. I am not completely partial to it so I fine with keeping the naming `keypath` in `StakeDelegation` certificate

Reply:
This is also what seemed to break CI:

```
    --> bitbox02-rust/src/hww/api/cardano/sign_transaction.rs:526:29
    |
526 | ...                   keypath: vec![1852 + HARDENED, 1815 + HARDENED, HARDENED, 2, 0],
    |                       ^^^^^^^ `StakeDelegation` does not have this field
    |
```

Reply:
I'd leave it as it was, as otherwise:

- renaming it here means that downstream libraries need to change and get a major version bump, and the Adalite integration with the bitbox-api NPM package then also needs to update the name.
- this update would seem to be inconsistent with the `stake_registration` and `stake_deregistration` fields, where the name is also still `keypath`.

Could it make sense to simply name the VoteDelegation keypath field `keypath` instead for consistency?

The TypeScript types right now are derived from the .proto field names like this:

```TypeScript
type CardanoCertificate =

  | {

      stakeRegistration: {

        keypath: Keypath

      }

    }

  | {

      stakeDeregistration: {

        keypath: Keypath

      }

    }

  | {

      stakeDelegation: {

        keypath: Keypath

        poolKeyhash: Uint8Array

      }

    };
```

Reply:
I think it can make sense, but I will add a comment above it so at least in code there is a comment that it is supposed to be `stake_keypath`